index.bs

```diff
@@ -1104,9 +1105,13 @@ values:
 Note: A version 4 (pseudorandom) [=UUID=] is recommended as it meets the
 requirements for a remote-playback-id.
 
-: urls
-:: The [=media resources=] that the controller has selected for playback on the
-    receiver.
+: sources The [=media resources=] that the controller has selected for playback on the
+    receiver.  Each source must include a URL
+    (see [HTMLSourceElement.src](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-source-src))
```

Comment:
Can you use Bikeshed autolinking, like {{HtmlMediaElement#src}}?

https://tabatkins.github.io/bikeshed/#autolink-microsyntax